barretenberg/cpp/src/barretenberg/commitment_schemes/commitment_key.test.hpp

```diff
@@ -39,10 +39,9 @@ template <>
 inline std::shared_ptr<VerifierCommitmentKey<curve::BN254>> CreateVerifierCommitmentKey<
     VerifierCommitmentKey<curve::BN254>>()
 {
-    constexpr size_t n = 4096;
     std::shared_ptr<bb::srs::factories::CrsFactory<curve::BN254>> crs_factory(
```

Comment:
is there a point to this line anymore?

Reply:
No, idk why the compiler didn't complain.